### ISSUE TITLE
Datatable: scrollable property loses column alignment

### ISSUE THREAD

**Issue by Dav3rs** [primefaces/primereact#3091]
### Describe the bug

This occurs just by adding the scrollable property when cell values are very different each others:
![image](https://user-images.githubusercontent.com/50024128/180228501-26a49e6a-0253-4f1b-978d-862ec6cc428c.png)

Just by removing the "scrollable" property:
![image](https://user-images.githubusercontent.com/50024128/180228541-a64f4bae-7c0a-4d81-b93b-6f2574d2fab7.png)

Check the next sandbox:
![image](https://user-images.githubusercontent.com/50024128/180228665-34334334-0c3f-478e-b728-48724211035a.png)

### Reproducer

https://codesandbox.io/embed/great-monad-pzyj6p?fontsize=14&hidenavigation=1&theme=dark

### PrimeReact version

8.3.0

### React version

18.x

### Language

TypeScript

### Build / Runtime

Next.js

### Expected behavior

Keep the column alignment.

**Comment by riyasat** [comment]
Hi,
There is also one more bug related to the scrollable. 
When you make datatable as scrollable you cannot set the column width. You can reproduce it on same link as mentioned in the bug report by adding style={{width:'3em'}}.

Regards

**Comment by ubau** [comment]
Simple workaround for this issue is to set min width for every column.
You can try it in same reproducer just by adding ```style={{minWidth: "100px"}}```

**Comment by melloware** [comment]
Excellent workaround @ubau !

**Comment by Dav3rs** [comment]
@melloware That is a workaround for the with style issue, but not for the main ticket issue, because if I set some harcoded min-width for the columns, but then some cells have a bigger lenght the columns are getting messy anyway.
In fact this min-width usage is on the official exaple docs
https://www.primefaces.org/primereact/datatable/scroll/
Another problem is that if the table has a large number of columns, min-width will cause unused empty spaces that cannot be abused.
Please don't drop priority over this issue🙏. It will be a big score for the next release 🚀

**Comment by melloware** [comment]
I don't work for PrimeTek. 

I assigned it to Mert who is in charge of this project for PrimeTek.

**Comment by ubau** [comment]
Well, this is just a workaround and issue still remains.
Speaking about unused empty spaces you can use different min width for every column. 
If the cell content is biger than column width there are two options - you can wrap content or use:
```className="white-space-nowrap overflow-hidden text-overflow-ellipsis"```  and show full content in Tooltip.

**Comment by melloware** [comment]
Looks like this is a duplicate this person had the same issues and I commented with solutions and screenshots: https://github.com/primefaces/primereact/issues/2951

**Comment by melloware** [comment]
I think this ticket is also related: https://github.com/primefaces/primereact/issues/2479 @ubau

**Comment by Dav3rs** [comment]
If this styles are removed from the row

![image](https://user-images.githubusercontent.com/50024128/180610740-7886c959-827d-4729-97e1-b1e0c84c1837.png)

This from the header

![image](https://user-images.githubusercontent.com/50024128/180611464-86cf4d11-9ab6-4a37-bb4b-427b2d4a7746.png)

And this styles are removed from the cells

![image](https://user-images.githubusercontent.com/50024128/180610760-97ff9af2-f44f-47e7-8036-5a2336cac8f8.png)

The bug is completely and beautifully gone.

It can be localy fixed by overwriting the styles with
```css
.p-datatable-scrollable .p-datatable-thead > tr,
.p-datatable-scrollable .p-datatable-tbody > tr,
.p-datatable-scrollable .p-datatable-tfoot > tr {
	display: table-row !important;
}
.p-datatable-scrollable .p-datatable-thead > tr > th,
.p-datatable-scrollable .p-datatable-tbody > tr > td,
.p-datatable-scrollable .p-datatable-tfoot > tr > td {
	display: table-cell !important;
}

.p-datatable-scrollable .p-datatable-thead,
.p-datatable-scrollable .p-datatable-tbody,
.p-datatable-scrollable .p-datatable-tfoot {
	display: table-header-group !important;
}
```

I confirm, the bug is gone.

**Comment by mertsincan** [comment]
Thanks a lot, @Dav3rs! We changed datatable's structure. I think this issue has already been fixed in 9.2.0. Please try it.
https://github.com/primefaces/primereact/issues/4078

Best Regards,

**Comment by melloware** [comment]
Confirmed this is fixed in 9.2.0
